### PR TITLE
Add NNPOps Again

### PR DESCRIPTION
**Pull request by mikemhenry**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mikemhenry** [comment]
Supersedes https://github.com/conda-forge/staged-recipes/pull/16257

File: recipes/NNPOps/build.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+CMAKE_FLAGS="${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release -DPython_EXECUTABLE=${PYTHON}"
+CMAKE_FLAGS+=" -DTorch_DIR=${SP_DIR}/torch/share/cmake/Torch"
+#CMAKE_FLAGS+=" -DENABLE_CUDA=OFF"
+
+mkdir build && cd build
+cmake ${CMAKE_FLAGS} ${SRC_DIR}
+make -j$CPU_COUNT
+make -j$CPU_COUNT install
+CTEST_OUTPUT_ON_FAILURE=1 ctest --verbose --exclude-regex TestCuda
```
Comment (jaimergp) [inline recipes/NNPOps/build.sh:53]: Reminder to change this in the feedstock for better testability downstream. (These tests should be built here, but run on `test`).

Comment (jaimergp) [reply recipes/NNPOps/build.sh:53]: xref https://github.com/conda-forge/staged-recipes/pull/16257#discussion_r760956583

Comment (mikemhenry) [reply recipes/NNPOps/build.sh:53]: Yes I will do that once we get it in the feedstock! I did try playing with that in the last PR and there were some issues in getting the files copied over to the right spot, which I will sort out later.

**Comment by mikemhenry** [comment]
@jaimergp It should be good to merge! Only failure is due to lack of diskspace

**Pull request merged**